src/components/Overview.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div id="overview">
+    <div class="row">
+      <div class="col-xs-4 col-md-12">
+        <h3>Last Meal</h3>
+        <span>{{lastMealName}}</span>
+      </div>
+
+      <div class="col-xs-4 col-md-12">
+        <h3>Most Prepared</h3>
+        <ul class="list-unstyled" v-for="(meal, i) in mostPrepared" :key="i">
+          <li>{{meal.name}}</li>
+        </ul>
+      </div>
+
+      <div class="col-xs-4 col-md-12">
+        <h3>Least Prepared</h3>
+        <ul class="list-unstyled" v-for="(meal, i) in leastPrepared" :key="i">
+          <li>{{meal.name}}</li>
+        </ul>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+  import {sortMostUsed, sortRecentlyPrepared} from"@/business/meals"
+  export default {
+    name: 'Overview',
+    props: ["meals"],
+    computed: {
```

Comment:
these computed properties turned out to be a nice place to format data for display. The actual saved values are managed outside of the view component and not affected by them.